pkg/pfcpsim/session/pfcp_session.go

```diff
@@ -0,0 +1,24 @@
+package session
```

Comment:
We should keep this in `pfcpsim` package - there is no need to create separate packages.